teams/raft/team.json

```diff
@@ -0,0 +1,28 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "BusyJay",
+        "hicqu"
+    ],
+
+    "committers": [
+        "5kbpers",
+        "NingLin-P",
+        "gengliqi",
+        "overvenus",
+        "zhouqiang-cl"
+    ],
+
+    "reviewers": [
+        "Fullstop000",
+        "accelsao"
```

Comment:
@BusyJay do you know who is @accelsao? It seems GitHub ID changes and I cannot find the user.

Reply:
OK. It turns to be @jayzhan211

Reply:
Bad idea for github to allow changing ID.

Reply:
```suggestion
        "jayzhan211"
```

Reply:
GitHub also has real digital IDs. A better solution is to use real IDs. It's available through the API, but it requires us to make some automated tools to help people get it automatically, otherwise the experience would be terrible.

Reply:
For example: https://github.com/rust-lang/team/blob/master/people/hi-rustin.toml